hollow-explorer-ui/src/main/java/com/netflix/hollow/explorer/ui/pages/HollowExplorerPage.java

```diff
@@ -43,12 +43,18 @@ public HollowExplorerPage(HollowExplorerUI ui) {
     public void render(HttpServletRequest req, HttpServletResponse resp, HollowUISession session) throws IOException {
         VelocityContext ctx = new VelocityContext();
 
-        if(ui.getHeaderDisplayString() != null)
-            ctx.put("headerDisplayString", ui.getHeaderDisplayString());
-        
-        if(ui.getCurrentStateVersion() != Long.MIN_VALUE)
+        if (ui.getCurrentStateVersion() != Long.MIN_VALUE)
             ctx.put("stateVersion", ui.getCurrentStateVersion());
-        
+
+        String headerDisplayString = ui.getHeaderDisplayString();
+        ctx.put("headerDisplayString", headerDisplayString);
+
+        if (headerDisplayString != null) {
+            String headerDisplayURL = ui.getFromHeaderDisplayMap(headerDisplayString);
+            headerDisplayURL = headerDisplayURL + "/" + headerDisplayString;
```

Comment:
is this intentional?

Reply:
Removed the URL creation part from Hollow. This decision now happens in the internal client.